packages/app/src/components/Synchronization/SynchronizationAPIProvider.tsx

```diff
@@ -15,34 +19,82 @@ type SynchronizationAPIProviderProps = PropsWithChildren<{
   accessToken: string;
 }>;
 
+export interface SynchronizationConfig {
+  isAuthorized: boolean;
+  authUrl?: string;
```

Comment:
does this need to be in the context? doesn't seem to be used anywhere outside the login function

Reply:
Nope. Leftover. Updated